src/lib/Preview.js

```diff
@@ -1423,14 +1423,18 @@ class Preview extends EventEmitter {
             .then(() => {
                 // Reset retry count after successfully logging
                 this.logRetryCount = 0;
```

Comment:
@bfoxx1906 actually i spotted one more bug on this line. this logRetryCount is reset to 0 every time after we execute logPreviewEvent again. so we need to remove this line in order to fix retry logic. because originally request should stop after logRetryCount reach the threshold of 3 retries but instead retry counter resets on every execution

Reply:
Not seeing where we reset it to 0? That said this looks like it should prevent the infinite recursion. Have we verified that there are no negative impacts when the event is emitted for non document previews? It probably is't a problem but just wanted to make sure we thought about that.

Reply:
I checked non document files and did not notice bugs with event. Here is video recording https://cloud.box.com/s/heb6e7c2g6u5ogmu0dbu9ny51dknq10f

@bfoxx1906 Let me know if there any more changes is needed